tests/test_common_features.py

```diff
@@ -15,10 +15,26 @@
 COMPONENT_PYTHON_BASE = 'python'
 COMPONENT_REACT_BASE = 'react'
 
+
+def bool_converter(s):
+    """Return the same as built-in function bool() except for arguments which are
+    string representations of a boolean value.
+
+    :param s: a variable
+    :return: True or False
+    representation of the boolean value
```

Comment:
@Bachibouzouk You want to remove this line.
I wasn't sure how to do it in suggestion mode, because blank line is not the same as no line...